library/std/src/f128/tests.rs

```diff
@@ -38,3 +38,9 @@ macro_rules! assert_f128_biteq {
         );
     };
 }
+
+#[test]
+fn test_neg_zero() {
+    // only verify const eval negation until we add the rest of the checks
+    let _neg_zero: f16 = -0.0;
+}
```

Comment:
These are thoroughly covered by the miri tests, I don't think we need it here, but if you wanna keep it, this needs to be f128

Reply:
Also this isn't even doing any const-eval? If anything it might be hitting some mir-opt codepaths.

Reply:
yea, const prop was hitting the mir interp code

Reply:
Ah thanks for the catch, I missed that it was double negation that caused the crash in #124583. Updated `tests/ui/numbers-arithmetic/f16-f128-lit.rs` instead